### DISCUSSION TITLE
Allow converting small projects into copy-pasteable URL's

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/extensions#1019] [Ideas]
This may **not** be implemented, but we should be able to embed our own project links on Turbowarp.
It would expand the uses of projects.

The project size limit is 50 MB. For copy-pasteable URL's, it should be Base64, and there should be a 75 KB (100000 characters) limit.
Like this: turbowarp.org/editor?project=**base64 data (100000 chars max)**

Let me know what you think.

**Comment by DNin01** [comment]
I think it's surprisingly easy to reach 75 KB with the way project files are encoded. Even easier with even a single costume or sound.

**Reply by ghost** [reply]
Didn't think of that lol.
